automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/FireCloudClient.scala

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.model.{Multipart, _}
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, _}
+import akka.util.ByteString
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import org.broadinstitute.dsde.firecloud.config.AuthToken
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor, Future}
+
+class FireCloudClient {
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  val mapper = new ObjectMapper()
+  mapper.registerModule(DefaultScalaModule)
+
+  // TODO: Is this okay here or should it be in a separate object and explicitly imported?
```

Comment:
I'm fine with it here: it will automatically be available to subclasses. Could mark it `protected` so _only_ subclasses can access it.